.travis.yml

```diff
@@ -40,14 +43,14 @@ before_script:
 script:
   # --scan enables the Gradle build scan, which can be used to investiage the time each action consumes
   # For more information see https://gradle.com/scans/get-started
-  - if [ "$TEST_SUITE" != "guiTest" ] && [ "$TEST_SUITE" != "checkstyle" ]; then ./gradlew $TEST_SUITE $OPTIONS --scan; fi
+  - if [ "$TEST_SUITE" != "checkstyle" ] && [ "$TEST_SUITE" != "guiTest" ] && [ "$TEST_SUITE" != "snapcraft" ]; then ./gradlew $TEST_SUITE $OPTIONS --scan; fi
   - if [ "$TEST_SUITE" == "checkstyle" ]; then ./gradlew checkstyleMain checkstyleTest checkstyleJmh; fi
   - if [ "$TEST_SUITE" == "guiTest" ]; then ./buildres/gui-tests.sh; fi
 
 after_success:
   # enable codecov report
-  - ./gradlew jacocoTestReport
-  - bash <(curl -s https://codecov.io/bash)
+  - if [ "$TEST_SUITE" != "checkstyle" ] && [ "$TEST_SUITE" != "snapcraft" ]; then ./gradlew jacocoTestReport && bash <(curl -s https://codecov.io/bash); fi
```

Comment:
not checkstyle and not snapcraft makes guitest and normal test... checkstyle does not run any tests and snapcroft doesn't too.